tests/compiletest.rs

```diff
@@ -99,18 +99,15 @@ fn test_config(target: &str, path: &str, mode: Mode, with_dependencies: bool) ->
     };
 
     if with_dependencies {
+        // Set the `cargo-miri` binary, which we expect to be in the same folder as the `miri` binary.
+        // (It's a separate crate, so we don't get an env var from cargo.)
+        let mut prog = miri_path();
+        prog.set_file_name("cargo-miri");
```

Comment:
This needs the `.exe` extension on Windows, right?

Reply:
Does it?

I guess CI will tell...

Reply:
.exe is in the PATHEXT env var so it should just execute (that's how my miri.bat file works). Though since this is executed in bash on msys2 I'm not sure if that applies or not as I do not use that.

Reply:
I thought PATHEXT only applied to cmd.exe and `ShellExecute` and not `CreateProcess`. `std::process::Command` uses `CreateProcessW`. The documentation of `CreateProcessW` doesn't mention whether or not it does apply

Reply:
Windows CI seems to be working so it does seem to add the `.exe`. :shrug:

Reply:
`Command`, for security reasons, uses its own logic for finding the exe and for compatibility reasons it always assumes `.exe` was intended if no extension is present.

Reply:
@ChrisDenton do you always show up when someone says the word "Windows" in a Rust repo? ;)

Anyway, thanks for the explanation! :)

Reply:
Yes, but only if the proper rituals are observed